modules/openapi-generator/src/main/resources/typescript-axios/api.mustache

```diff
@@ -191,7 +191,7 @@ export const {{classname}}AxiosParamCreator = function (configuration?: Configur
                 })
             {{/isCollectionFormatMulti}}
             {{^isCollectionFormatMulti}}
-                    localVarFormParams.set('{{baseName}}', {{paramName}}.join(COLLECTION_FORMATS["{{collectionFormat}}"]));
+                    localVarFormParams.set('{{baseName}}', {{paramName}}.join(COLLECTION_FORMATS["{{collectionFormat}}"]);
```

Comment:
I think this should be reverted, now one closing bracket `)` is missing

Reply:
oh right thanks

Reply:
pushed via --force thanks for the hint @macjohnny